djstripe/stripe_objects.py

```diff
@@ -1097,6 +1187,61 @@ def _stripe_object_to_charge(cls, target_cls, data):
         if "charge" in data and data["charge"]:
             return target_cls._get_or_create_from_stripe_object(data, "charge")[0]
 
+    @classmethod
+    def upcoming(cls, api_key=settings.STRIPE_SECRET_KEY, **kwargs):
```

Comment:
Also, since this is for a specific customer, add this to Customer as an instance method -- maybe ``Customer.upcoming_invoice()``. Follow the convention of returning the stripe object in stripe_objects.py and returning the dj-stripe model instance in models.py.